Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Parent Dockerfile https://github.com/docker-library/mongo/blob/982328582c74dd2f0a9c8c77b84006f291f974c3/3.0/Dockerfile
+# Inspired by: http://stackoverflow.com/questions/33558506/how-to-create-a-mongo-docker-image-with-default-collections-and-data?answertab=votes#tab-top
+FROM mongo:3.4
+
+# Modify child mongo to use /data/db2 as dbpath (because /data/db wont persist the build)
+RUN mkdir -p /data/db2 \
+    && echo "dbpath = /data/db2" > /etc/mongodb.conf \
```

Comment:
It should be possible to create the `mongodb.conf` in this repo and copy it into the container rather than creating it via the `RUN` command